packages/theming/atlas/src/themesource/atlas_core/native/core/widgets/radiobuttons.ts

```diff
@@ -18,36 +18,46 @@ const RADIO_BUTTON_INNER_CIRCLE_SIZE = 8;
 
 export const com_mendix_widget_native_radiobuttons_RadioButtons: RadioButtonsStyles = {
     labelTextStyle: {
+        // All TextStyle properties are allowed
         color: font.colorTitle,
         fontSize: font.sizeSmall,
         lineHeight: font.lineHeightSmall,
         marginBottom: spacing.small
     },
     radioButtonItemContainerStyle: {
+        // All ViewStyle properties are allowed
         marginBottom: spacing.small
     },
     radioButtonItemContainerHorizontalStyle: {
+        // All ViewStyle properties are allowed
         marginEnd: spacing.small
     },
     circularButtonStyle: {
+        // width, height, borderRadius, borderColor and All ViewStyle properties are allowed
```

Comment:
If im not mistaken, border related styles _are_ `ViewStyle`s? If so, then you can simplify. Also capitalision is off

Reply:
done